lms/envs/common.py

```diff
@@ -2677,3 +2677,6 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+# Grade book: max students per page
+MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 25
```

Comment:
this is also probably not the right place for this setting to live. We may also want to just pick a number and not allow it to be site configurable, because this configuration file is 2682 (!!!!) lines long, which is insane.